src/nls/ja/strings.js

```diff
@@ -160,7 +160,7 @@ define({
 	"FIND_NO_RESULTS": "該当なし",
 	"FIND_QUERY_PLACEHOLDER": "検索\u2026",
 	"FIND_HISTORY_MAX_COUNT": "検索履歴内の検索項目の最大数",
-	"REPLACE_PLACEHOLDER": "\u2026 に置換",
+	"REPLACE_PLACEHOLDER": " に置換\u2026",
```

Comment:
There are two PR for the same thing or any change is there? Can you please close one PR.